src/providers/ldap/sdap.c

```diff
@@ -648,10 +648,13 @@ static bool objectclass_matched(struct sdap_attr_map *map,
         len = strlen(objcl) + 1;
     }
 
-    if (strncasecmp(map[SDAP_OC_GROUP].name, objcl, len) == 0) {
+    if (strncasecmp(map[0].name, objcl, len) == 0) {
```

Comment:
I am confused. At this point you decide to remove the reference to SDAP_OC_OC_GROUP and put its value in its place, but later in the code you still use references to SDAP_OC_GROUP*. Why do you do this?

Reply:
Hi,

I guess this is a bit due to my comments. `objectclass_matched()` is not only used for groups and the first check will check the objetclass of any object, so using `SDAP_OC_GROUP` might be confusing and since there is no generic macro for the objectclass it is ok to use just `0` here.

The second check is specific for group objects because only groups might have an alternative object class, so it make sense to keep the group specific  `SDAP_OC_GROUP_ALT`.

HTH

bye,
Sumit

Reply:
Ok, thanks for the explanation Sumit